.circleci/config.yml

```diff
@@ -1477,7 +1477,7 @@ jobs:
           name: Propagate names for AWS essential components
           command: |
             sed -i -e '/^secgroupDefault: /s/:.*$/: publish-amis-to-marketplace-secgroup/' $VARS_DIR/defaults.yaml
-            sed -i -e '/^bucketDefault: /s/:.*$/: publish-amis-to-marketplace-bucket/' $VARS_DIR/defaults.yaml
+            sed -i -e '/^bucketDefault: /s/:.*$/: publish-amis-to-marketplace-bucket2/' $VARS_DIR/defaults.yaml
```

Comment:
Since bucket names are globally unique, is there a way to add a check that validates if bucket already exists and has write permissions for the circle-ci user?

Reply:
Just added  existence and write permission check for the export bucket but not for the default bucket.
The default bucket is created by the playbooks and playbooks will fail if it already exists.

Reply:
LGTM.